### PR TITLE
adds moped_proj_contract to activity log

### PR DESCRIPTION
**Pull request by tillyw**
## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/10808

## Testing
**URL to test:** 

https://10808_moped_proj_contract_activity_log--atd-moped-main.netlify.app/moped/

**Steps to test:**
- add a contract with a contractor name
- add a contract without a contractor name
- edit the other fields on the contract with a contractor name
- edit the other fields on the contract without a contractor name
- delete the contract with a contractor name
- delete the contract without a contractor name
- the activity log should reflect all six of these actions in unique ways

---
#### Ship list
- [x] Code reviewed 
- [x] Product manager approved
- [x] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/views/projects/projectView/ProjectActivityLog.js [hunk 1]
```diff
@@ -321,7 +321,8 @@ const ProjectActivityLog = () => {
                           "moped_proj_notes",
                           "moped_proj_components",
                           "moped_project_types",
-                          "moped_proj_personnel"
+                          "moped_proj_personnel",
+                          "moped_proj_contract",
```
Comment (tillyw) [inline moped-editor/src/views/projects/projectView/ProjectActivityLog.js:325]: sticking with this pattern for the time being!

**Comment by tillyw** [comment]
please hold off on reviewing - there's something I need to check on!

**Comment by tillyw** [comment]
good to go!

**Review by chiaberry — COMMENTED** [review]
I was curious why you deployed this via the moped test commission flow, but then I checked the auto netlify deploy preview (https://deploy-preview-971--atd-moped-main.netlify.app/moped/projects/1900?tab=activity_log) so I could see how it looked with all of the staging data, and I noticed its not working there. 

The activities are showing up for me locally and in your preview, so I am thinking its something isolated to the netlify preview? I'm going to check

**Comment by tillyw** [comment]
resolved conflicts with main!

**Review by roseeichelmann — APPROVED** [review]
this looks and works great! love seeing the activity log coming together ✨✨

**Review by chiaberry — APPROVED** [review]

**Review by mddilley — APPROVED** [review]
Tested all the steps, and they all ✅ out. 🙌 Once again, this pattern handles variations like contractor name/no contractor name beautifully! 🚢

File: moped-database/metadata/tables.yaml [hunk 2]
```diff
@@ -1845,6 +1845,56 @@
...
+          value_from_env: ACTIVITY_LOG_API_SECRET
+      request_transform:
+        body:
+          action: transform
+          template: |-
+            {
+              "query": "mutation InsertActivity($object: moped_activity_log_insert_input!, $project_id:Int!, $updated_at:timestamptz ) { insert_moped_activity_log_one(object: $object) { activity_id } update_moped_project_by_pk(pk_columns: {project_id: $project_id}, _set: {updated_at: $updated_at}) { updated_at }}",
+              "variables": {
+                "object": {
+                  "record_id": {{ $body.event.data.new.id }},
```
Comment (johnclary) [inline moped-database/metadata/tables.yaml:1874]: 💯 👍

**Review by johnclary — APPROVED** [review]
sorry for the noise! i was getting a build error b/c of the #973, so i merged main and pushed. but then i realized i had accidentally committed to main the mock-ups i did for [#11428](https://github.com/cityofaustin/atd-data-tech/issues/11428). so i reverted that commit. yeesh.


@tillyw—this looks perfect! i'm very happy to have this data being tracked now 🚀 🚢 🚢 🚢 🚢 🚢 🚢 🚢 🚢

**Pull request merged**